packages/core/ai_content_platform/vimax/cli/commands.py

```
@@ -332,20 +336,72 @@ def generate_storyboard(script, output, image_model, style, portraits, reference
+    ref_shots = [
+        shot
+        for scene in script_obj.scenes
+        for shot in scene.shots
+        if shot.character_references or shot.primary_reference_image
+    ]
+    if ref_shots or portrait_registry:
+        label = "Resolved" if portrait_registry else "Inline"
+        if ref_shots:
+            click.echo(f"\n   {label} references for {len(ref_shots)} shots:")
+        else:
+            click.echo(f"\n   No character references matched from portrait registry.")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)